### DISCUSSION TITLE
I tried sending Serum Eth, to Serum Solana on Wormhole! Please help? Token missing?

### DISCUSSION THREAD

**Discussion by marceckosage** [wormhole-foundation/wormhole#408] [Problems / bugs] [answered]
I tried sending Serum Eth, to Serum Solana on Wormhole! Please help? Token missing?

The token address is showing up as an UNKNOWN token?

address: 2jXy799YnEcRXneFo2GEAB6SDRsAa767HpWmktRr1DaP

https://solscan.io/token/2jXy799YnEcRXneFo2GEAB6SDRsAa767HpWmktRr1DaP

Did I just loose all of that Serum???

**Comment by marceckosage** [comment]
https://solanabeach.io/address/2jXy799YnEcRXneFo2GEAB6SDRsAa767HpWmktRr1DaP

Above is Token address?

**Comment by leoluk** [comment, marked as answer]
Please provide the transaction

**Reply by marceckosage** [reply]
https://explorer.solana.com/tx/2Bju1H7LddGaqcRjQoypdc7voqJf4LFWPUJYEMm68ctcLMJVpNdMfNRn48m5yB3g5EF2yMiQue9RmZrvcNBLaCPe

sig: 2Bju1H7LddGaqcRjQoypdc7voqJf4LFWPUJYEMm68ctcLMJVpNdMfNRn48m5yB3g5EF2yMiQue9RmZrvcNBLaCPe

Sep 7, 2021 at 23:01:40 Eastern Daylight Time

block: 95496446

Block Hash: 7tQNjafZsgRZUoBGy4sWyVaY1Leqv9vKU7uNcS9CpQud

Program Log:
Program WormT3McKhFJ2RkiGpdw9GKvNCrB2aB54gb2uV9MfQC invoke [1]
Instruction: PostVAA
Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA invoke [2]
Instruction: MintTo
Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA consumed 2779 of 104298 compute units
Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA success
deriving key
deploying contract
Program 11111111111111111111111111111111 invoke [2]
Program 11111111111111111111111111111111 success
Program WormT3McKhFJ2RkiGpdw9GKvNCrB2aB54gb2uV9MfQC consumed 132543 of 200000 compute units
Program WormT3McKhFJ2RkiGpdw9GKvNCrB2aB54gb2uV9MfQC success

**Reply by marceckosage** [reply]
Thank you for you attention Leo 🙏🏻

**Reply by marceckosage** [reply]
> Please provide the transaction

Any feedback? Suggestions for a mend? @leoluk

**Reply by leoluk** [reply]
According to that transaction, the transfer worked and you now have wrapped SRM.

**Reply by leoluk** [reply]
(`2jXy799YnEcRXneFo2GEAB6SDRsAa767HpWmktRr1DaP` is indeed wSRM - Saber has a pool, for instance)

**Reply by marceckosage** [reply]
> Please provide the transaction

I am uncertain how this may be marked as answered? Is there something I should technically do to revert transaction? Or is it missing? Sorry to trouble you, I see you have been nobly trying to help many people on this issue. Thank you.

**Reply by marceckosage** [reply]
<img width="353" alt="Screenshot_9_11_21__10_49_AM" src="https://user-images.githubusercontent.com/76853936/132951840-336547d4-078a-4463-b8f0-e1a7badfecbc.png">
> (`2jXy799YnEcRXneFo2GEAB6SDRsAa767HpWmktRr1DaP` is indeed wSRM - Saber has a pool, for instance)

@leoluk thank you for answering this.  I would like to send you some tokens for you patience and generous sharing of information.  I have taken the "token address: 2jXy799YnEcRXneFo2GEAB6SDRsAa767HpWmktRr1DaP" and tried entering that in my wallet.  I do not see a wSRM token within the market. I am on the raydium dex, and do not see it there either.  When I sign in using my Phantom wallet, the dex does NOT recognize or offer any wSRM token.  How can I exchange it into regular SRM within the solana ecosystem?  If you help me retreive, I will gladly share some for your valuable time and guidance. With Gratitude, ME.

**Reply by marceckosage** [reply]
@leoluk You saved the day! I had never even heard of saber until you mentioned it here! I owe you! Please send me an address and let me buy you a drink for your time! I am loving solana!!!!

**Reply by leoluk** [reply]
No need - glad it worked out!
